app/src/main/res/values-hi/strings.xml

```diff
@@ -54,7 +54,10 @@
   <string name="gps_disabled">आपके डिवाइस में जीपीएस अक्षम है। क्या आप इसे सक्षम करना चाहेंगे?</string>
   <string name="enable_gps">जीपीएस सक्षम करें</string>
   <string name="contributions_subtitle_zero">अभी तक कोई अपलोड नहीं</string>
-  <string name="contributions_subtitle">{{PLURAL | शून्य = @ स्ट्रिंग / योगदान_उपशीर्षक_शुन्य | एक =% d अपलोड |% d अपलोड्स}}</string>
+  <plurals name="contributions_subtitle">
+    <item quantity="one">%d अपलोड</item>
+    <item quantity="other">%d अपलोड्स</item>
+  </plurals>
```

Comment:
Is this related?

Reply:
I'm not sure what happened here, I rebased to latest and got failing builds, so I needed to fix this in order to continue. Looks like a translation update added a spurious string entry...